src/style.css

```diff
@@ -0,0 +1,47 @@
+html {
+  margin: 0;
+}
+
+body {
+  margin: 0;
+}
+
+.star {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--0 .stars__star {
+  background-image: url(images/star.svg);
+}
+
+.stars--1 .stars__star:nth-child(1) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--2 .stars__star:nth-child(-n+2) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--3 .stars__star:nth-child(-n+3) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--5 .stars__star:nth-child(n) {
+  background-image: url(images/star-active.svg);
+}
```

Comment:
[checklist](https://github.com/mate-academy/layout_stars/blob/master/checklist.md) rule number 2

Reply:
Good night. I don't  understand how to apply this rule correctly. Should it look something like this?
[https://imgur.com/a/n7bS7ko](url)